### DISCUSSION TITLE
zhimg问题

### DISCUSSION THREAD

**Discussion by PGXx** [AlistGo/alist#411] [Q&A]
坐标北京，不知道是我个人原因还是怎么回事，我在访问作者您的演示站点，还有我自己的站点时，无法加载图片中的JS文件。
我的服务器是腾讯香港轻量云，在2.0.6版本的时候没有问题，今天更新到2.0.7后出现的这个错误，我用的是一键脚本搭建的。
![20220121203905](https://user-images.githubusercontent.com/54702820/150528777-634042c4-7b4f-4320-b93e-abc689ae2079.png)

**Comment by xhofe** [comment]
https://alist-doc.nn.ci/docs/setting/config#assets
